packages/lib/import-enex-html-gen.js

```diff
@@ -121,9 +122,11 @@ function enexXmlToHtml_(stream, resources) {
 					section.lines = addResourceTag(section.lines, resource, nodeAttributes);
 				}
 			} else if (tagName === 'en-todo') {
-				const nodeAttributes = attributeToLowerCase(node);
 				const checkedHtml = nodeAttributes.checked && nodeAttributes.checked.toLowerCase() === 'true' ? ' checked="checked" ' : ' ';
 				section.lines.push(`<input${checkedHtml}type="checkbox" onclick="return false;" />`);
+			} else if (tagName === 'li' && nodeAttributes?.style?.includes('--en-checked:')) {
```

Comment:
Remove the "?s". We need to know what data we're working with, not just assume everything's broken all the time. If something's undefined we need to know why and fix it, not just ignore it with "?".